Code/Framework/AzToolsFramework/AzToolsFramework/ComponentMode/ComponentModeSwitcher.cpp

```diff
@@ -295,7 +294,7 @@ namespace AzToolsFramework::ComponentModeFramework
         // if the user enters component mode not using the switcher, the switcher should still switch buttons
         if (mode == ViewportEditorMode::Component)
         {
-             if (auto componentDataIt = AZStd::find_if(
+            if (auto componentDataIt = AZStd::find_if(
```

Comment:
Future: [AZStd::ranges::find_if](https://en.cppreference.com/w/cpp/algorithm/ranges/find) can be used to only need to specify the [range container](https://github.com/o3de/o3de/blob/development/Code/Framework/AzCore/Tests/AZStd/RangesAlgorithmTests.cpp#L136-L151).

Reply:
Thank you for your review :) I didn't realize ranges were implemented, I've changed my usage to this!